maintenance/Maintenance.php

```diff
@@ -886,6 +888,10 @@ public function finalSetup() {
 		# Same with these
 		$wgCommandLineMode = true;
 
+		# MaintenanceStatus comes from a separate file (listed in AutoLoader.php) and we need to set it
+		# here as `finalSetup` is called after AutoLoader is included.
+		$this->status = new MaintenanceStatus($this->getName());
```

Comment:
this code should be spacey, you can get an authoritative IntelliJ code style XML from https://github.com/Wikia/guidelines/blob/master/PHP/bin/MediaWiki.xml